packages/pn-personagiuridica-webapp/src/component/Deleghe/ProxiesChargedCompany.tsx

```diff
@@ -0,0 +1,144 @@
+import { useTranslation } from 'react-i18next';
```

Comment:
I should rename the file as DelegationsOfTheCompany

Reply:
done